src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -1,14 +1,63 @@
 package codeu.controller;
 
 import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
 
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-public class ActivityFeedServlet extends HttpServlet{	
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-  	response.getOutputStream().println("This is the Activity Feed");	
-  }
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+
+public class ActivityFeedServlet extends HttpServlet {
```

Comment:
It looks like throughout this file you've indented 4 spaces instead of 2.

Reply:
You'll need to do this everywhere. 
e.g 
public void init() throws ServletException {
 		super.init();
                ....

should be 

public void init() throws ServletException {
  super.init();
  ....